src/ATen/native/xpu/WeightNorm.cpp

```diff
@@ -0,0 +1,27 @@
+#include <ATen/native/xpu/sycl/WeightNormKernels.h>
+#include <ATen/xpu/XPUNativeFunctions.h>
+namespace at {
+std::tuple<Tensor, Tensor> XPUNativeFunctions::_weight_norm_interface(
+    const Tensor& v,
+    const Tensor& g,
+    int64_t dim) {
+  return native::xpu::_weight_norm_interface_kernel(v, g, dim);
```

Comment:
`_` is meaningless for us, but meaningful for ATen. It means the operator is a low level / internal operator. Please remove it.
`weight_norm_kernel` is good to me.